pages/tips.md

```diff
@@ -16,26 +16,27 @@ This section contains **user-submitted tips'n tricks** on using JHipster.
 _If you want to contribute, don't hesitate to send us a Pull Request with your tips on our [GitHub repository](https://github.com/jhipster/jhipster.github.io)._
 
 1. [Create a static Swagger API documentation]({{ site.url }}/tips/008_tips_static_swagger_docs.html)
-1. [Using Bootswatch themes]({{ site.url }}/tips/009_tips_using_bootswatch_themes.html)
-1. [Configuring Email with - Gmail and more]({{ site.url }}/tips/011_tip_configuring_email_in_jhipster.html)
-1. [Speed up the generator-jhipster]({{ site.url }}/tips/013_tip_speed_up_generator.html)
-1. [Local SMTP Server]({{ site.url }}/tips/015_tip_local_smtp_server.html)
-1. [LDAP Authentication]({{ site.url }}/tips/016_tip_ldap_authentication.html)
-1. [REPL with the remote shell]({{ site.url }}/tips/017_tip_repl_with_the_remote_shell.html) (deprecated)
-1. [Kubernetes and Google Cloud SQL]({{ site.url }}/tips/018_tip_kubernetes_and_google_cloud_sql.html)
-1. [Boost performance of pagination with infinite scrolling using Slice]({{ site.url }}/tips/019_tip_infinite_scroll_with_slice.html)
-1. [Using Docker containers as localhost on Mac/Windows]({{ site.url }}/tips/020_tip_using_docker_containers_as_localhost_on_mac_and_windows.html)
-1. [Use QueryDSL]({{ site.url }}/tips/003_tip_add_querydsl_support.html)
-1. [Protecting Kibana with Apache (Basic Authentication)]({{ site.url }}/tips/023_tip_protecting_kibana_with_apache_basic_authent.html)
-1. [Enabling Social Login with OAuth2]({{ site.url }}/tips/024_tip_enabling_social_login_with_oauth2.html)
-1. [How to create a new Authority]({{ site.url }}/tips/025_tip_create_new_authority.html)
-1. [Issue of @OneToOne with @MapsId and how to avoid it]({{ site.url }}/tips/026_tip_issue_of_onetoone_with_mapsid_how_to_avoid_it.html)
-1. [Increase integration test performance by lazy bean initialization]({{ site.url }}/tips/027_tip_lazy_init_test_beans.html)
-1. [How to connect PGAdmin (PostgreSQL) to Heroku]({{ site.url }}/tips/028_tip_pgadmin_heroku.html)
-1. [Provide Internet Explorer support]({{ site.url }}/tips/028_tip_ie_support.html)
-1. [Improve developer experience if opening only front end in the IDE]({{ site.url }}/tips/029_tip_frontend_only.html)
-1. [Configure Redis leader follower(master-slave) replication]({{ site.url }}/tips/030_tip_redis_replication.html)
-1. [Running Protractor e2e tests within Intellij IDEA]({{ site.url }}/tips/031_tip_e2e_intellij.html)
-1. [Time Drift in Docker]({{ site.url }}/tips/032_tip_time_drift_docker.html)
-1. [User entity management]({{ site.url }}/user-entity/)
-1. [Remove the register account service]({{ site.url }}/tips/034_tip_remove_register_account_service.html)
+2. [Using Bootswatch themes]({{ site.url }}/tips/009_tips_using_bootswatch_themes.html)
```

Comment:
@Tcharl Rollback those changes, not needed to increment the number. see https://www.jhipster.tech/tips/

Reply:
Ok but why rollbacking? both works, and I prefer the second one, (and my and most of the IDE too, as they're automatically increasing the bit...)

Reply:
The reason for keeping all the numbers labelled "- 1." is because it is less change, and a smaller diff will be stored by git.
If the numbers were stored in ascending order and someone needed to insert a new entry after #12 then all of the following numbers would need to be renumbered again causing a bigger diff to be stored by git.
Smaller diff's are easier to review to see where change has been made.